packages/application-shell/src/components/navbar/menu-items.tsx

```diff
@@ -270,6 +271,7 @@ const MenuItem = (props: MenuItemProps) => {
       isActive={props.isActive}
       isRouteActive={Boolean(props.isMainMenuRouteActive)}
       isCollapsed={!props.isMenuOpen}
+      onMouseMove={props.onMouseMove as MouseEventHandler<HTMLLIElement>}
```

Comment:
Is the casting really needed? I tried removing it locally and I am not getting any squiggly lines.

Reply:
updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3536/commits/4055a9cbc064d49ac4c90ec42d85b55cf0b284b4
Thank you.